### PR TITLE
Adherenece calendar followup

### PR DESCRIPTION
**Pull request by emord**
https://github.com/dimagi/commcare-hq/pull/12923#issuecomment-241537408

@proteusvacuum

### CODE REVIEW

File: corehq/apps/app_manager/tests/test_advanced_suite.py [hunk 1]
```diff
@@ -177,6 +178,25 @@ def test_advanced_suite_load_case_from_fixture_with_arbitrary_datum(self):
                                    './entry[2]/session')
         self.assertXmlPartialEqual(self.get_xml('load_case_from_fixture_instance'), suite, './entry[2]/instance')
 
+    def test_advanced_suite_load_case_from_fixture_with_custom_fixture(self):
+        app = Application.wrap(self.get_json('suite-advanced'))
+        app.get_module(1).get_form(0).actions.load_update_cases.append(LoadUpdateAction(
+            case_tag="adherence",
+            case_type="clinic",
+            load_case_from_fixture=LoadCaseFromFixture(
+                fixture_nodeset="instance('enikshay:calendar')/calendar/year/month/day[@date > 735992 and @date < 736000]",
```
Comment (dimagimon) [inline corehq/apps/app_manager/tests/test_advanced_suite.py]: E501 line too long (123 > 115 characters)

File: corehq/apps/app_manager/tests/test_advanced_suite.py [hunk 2]
```diff
@@ -177,6 +178,25 @@ def test_advanced_suite_load_case_from_fixture_with_arbitrary_datum(self):
...
+                fixture_tag="selected_date",
+                fixture_variable="date",
+                case_property="adherence_event_date",
+                auto_select=True,
+            )
+        ))
+        with patch('corehq.apps.app_manager.suite_xml.post_process.instances.toggles.CUSTOM_CALENDAR_FIXTURE.enabled') as mock_toggle:
```
Comment (dimagimon) [inline corehq/apps/app_manager/tests/test_advanced_suite.py]: E501 line too long (134 > 115 characters)

File: corehq/apps/app_manager/tests/test_advanced_suite.py [hunk 3]
```diff
@@ -177,6 +178,25 @@ def test_advanced_suite_load_case_from_fixture_with_arbitrary_datum(self):
...
+            mock_toggle.return_value = True
+            suite = app.create_suite()
+        self.assertXmlPartialEqual(self.get_xml('load_case_from_custom_fixture_session'), suite, './entry[2]/session')
```
Comment (dimagimon) [inline corehq/apps/app_manager/tests/test_advanced_suite.py]: E501 line too long (118 > 115 characters)

File: corehq/apps/app_manager/tests/test_advanced_suite.py [hunk 4]
```diff
@@ -177,6 +178,25 @@ def test_advanced_suite_load_case_from_fixture_with_arbitrary_datum(self):
...
+        self.assertXmlPartialEqual(self.get_xml('load_case_from_custom_fixture_instance'), suite, './entry[2]/instance')
```
Comment (dimagimon) [inline corehq/apps/app_manager/tests/test_advanced_suite.py]: E501 line too long (120 > 115 characters)

**Comment by proteusvacuum** [comment]
Awesome, thanks! 👍 on ✅

**Pull request merged**